recipe/meta.yaml

```diff
@@ -42,12 +42,39 @@ test:
     - ipython3 -h          # [py3k]
   imports:
     - IPython
+    - IPython
+    - IPython.core
+    - IPython.core.magics
+    - IPython.core.tests
+    - IPython.extensions
+    - IPython.extensions.tests
+    - IPython.external
+    - IPython.external.decorators
+    - IPython.lib
+    - IPython.lib.tests
+    - IPython.sphinxext
+    - IPython.terminal
+    - IPython.terminal.pt_inputhooks
+    - IPython.terminal.tests
+    - IPython.testing
+    - IPython.testing.plugin
+    - IPython.testing.tests
+    - IPython.utils
+    - IPython.utils.tests
 
 about:
   home: https://ipython.org/
   license: BSD 3-clause
+  license_family: BSD
   license_file: COPYING.rst
   summary: "IPython: Productive Interactive Computing"
+  home: http://ipython.org/
```

Comment:
With the recent change of Chrome marking http:// as insecure, can we update that (and doc_url) to use https:// here and on the anaconda side ?